packages/server/src/queue/queue.entity.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { Connection } from "typeorm";
+import { TestingModule, Test } from "@nestjs/testing";
+import { TestTypeOrmModule } from "../../test/util/testUtils";
+import { QueueFactory, QuestionFactory } from "../../test/util/factories";
+import { QueueModel } from "./queue.entity";
+import { OpenQuestionStatus, ClosedQuestionStatus } from "@koh/common";
+
+describe('queue entity', () => {
+    let conn: Connection;
+  
+    beforeAll(async () => {
+      const module: TestingModule = await Test.createTestingModule({
+        imports: [TestTypeOrmModule],
+      }).compile();
+  
+      conn = module.get<Connection>(Connection);
+    });
+  
+    afterAll(async () => {
+      await conn.close();
+    });
+  
+    beforeEach(async () => {
+      await conn.synchronize(true);
+    });
+  
+      it('handles queueSize correctly', async () => {
+        const queueFactory = await QueueFactory.create();
+
+        await QuestionFactory.create({
+            queue: queueFactory,
+            status: OpenQuestionStatus.Queued
+        });
+        await QuestionFactory.create({
+            queue: queueFactory,
+            status: OpenQuestionStatus.Drafting
+        });       
+        await QuestionFactory.create({
+            queue: queueFactory,
+            status: OpenQuestionStatus.Helping
+        });       
+        await QuestionFactory.create({
+            queue: queueFactory,
+            status: ClosedQuestionStatus.Resolved
+        });
```

Comment:
super nitpick, but I think there's a `QuestionFactory.createList` that we could use

Reply:
I don't think the `.createList` function allows us to create a list of questions each with different attributes. 

https://github.com/antonversal/typeorm-factory/blob/ebc7a1a1c70f8053dda7a5225d26d1a152715157/src/Factory.ts#L139-L147